manager/integration/tests/test_basic.py

```diff
@@ -642,6 +642,42 @@ def test_backup_block_deletion():  # NOQA
     """
     pass
 
+
+@pytest.mark.skip(reason="TODO")
+def test_backup_volume_list():  # NOQA
+    """
+    Test backup volume list
+
+    Context:
+
+    We want to make sure that an error when listing a single backup volume
+    does not stop us from listing all the other backup volumes. Otherwise a
+    single faulty backup can block the retrieval of all known backup volumes.
+
+    Setup:
+
+    1. Setup NFS backupstore since we can manipulate the content easily
+
+    Steps:
+
+    1.  Create a volume(1) and attach to the current node
+    2.  Create a volume(2) and attach to the current node
+    3.  write some data to volume(1) & volume(2)
+    4.  Create a backup of volume(1) & volume(2)
```

Comment:
This looks good. But we might want to create multiple backups of volume(2), just to be sure backup list contains all the backups created.

Reply:
Good idea, I expanded the test steps to check for the existence of the backup information.